x-pack/plugins/synthetics/server/synthetics_service/project_monitor_formatter.ts

```diff
@@ -228,15 +260,78 @@ export class ProjectMonitorFormatter {
     return savedObjects?.[0];
   };
 
-  private createMonitor = async (normalizedMonitor: BrowserFields) => {
-    await syncNewMonitor({
-      normalizedMonitor,
-      monitor: normalizedMonitor,
-      server: this.server,
-      syntheticsMonitorClient: this.syntheticsMonitorClient,
-      savedObjectsClient: this.savedObjectsClient,
-      request: this.request,
-    });
+  private createMonitorsBulk = async (monitors: BrowserFields[]) => {
+    try {
+      if (monitors.length > 0) {
+        const { newMonitors } = await syncNewMonitorBulk({
+          normalizedMonitors: monitors,
+          server: this.server,
+          syntheticsMonitorClient: this.syntheticsMonitorClient,
+          soClient: this.savedObjectsClient,
+          request: this.request,
+          privateLocations: this.privateLocations,
+          spaceId: this.spaceId,
+        });
+
+        if (newMonitors && newMonitors.length === monitors.length) {
+          this.createdMonitors.push(...monitors.map((monitor) => monitor[ConfigKey.JOURNEY_ID]!));
+          this.handleStreamingMessage({
+            message: `${monitors.length} monitor${
+              monitors.length > 1 ? 's' : ''
+            } created successfully.`,
+          });
+        } else {
+          this.failedMonitors.push({
+            id: `${monitors.length}`,
```

Comment:
Set this as undefined, since this error is not related to a particular monitor. Feel free to update the type if necessary.